File: years/2022/clojure/aoc-2022/test/aoc_2022/day_02_test.clj
(ns aoc-2022.day-02-test
  (:require [aoc-2022.day-02 :refer :all]
            [clojure.test :refer :all]))

(deftest day-02-test
  (is (= 15 (part1 "../../data/02-sample.txt")))
  (is (= 12 (part2 "../../data/02-sample.txt"))))

File: years/2016/elixir/test/aoc/day_02_test.exs
defmodule AoC.Day02.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert AoC.Day02.part_1(["../data/02.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert AoC.Day02.part_2(["../data/02.txt"]) == 1
    end
  end
end
